.github/workflows/examples.yml

```diff
@@ -4,9 +4,13 @@ on:
   push:
     paths:
       - "glass/**"
+    branches:
+      - main
   pull_request:
     paths:
       - "glass/**"
+    branches:
+      - main
```

Comment:
In the event we are merging to non-`main` branches, are we sure we want this line?
```suggestion
```

Reply:
The main test workflow skips PRs on non-main branches too -

https://github.com/glass-dev/glass/blob/d7bd1e2f76bea37c67050c6acdae06fb18cfc71f/.github/workflows/test.yml#L7-L9

I guess it will make sense to remove it from there as well.

Reply:
Agreed